lib/modules/announcements.js

```diff
@@ -199,7 +197,8 @@ function isPostRelevant(post) {
 		return true;
 	}
 
-	if (postTitle.search(`${currentBrowser}`) > -1) {
+
+	if (postTitle.search(`${BrowserDetect.browser}`) > -1) {
```

Comment:
Sorry, I didn't quite follow your first comment. Are you suggesting removing the unnecessary variable assignment? I'm totally cool with that -  just making sure I'm not blind and missing something else. :)